features/support/env.rb

```diff
@@ -19,25 +33,33 @@
   # Create the local repository
   clone_repository :origin, :developer
 
-  # Create the main branch
-  in_repository :developer do
-    run 'touch .gitignore ; git add .gitignore ; git commit -m "Initial commit"; git push -u origin master'
-    run 'git checkout -b main master ; git push -u origin main'
-  end
-
   # Set main as the default branch
   in_repository :origin do
     run 'git symbolic-ref HEAD refs/heads/main'
   end
 
-  # Fetch the default branch, delete master
   in_repository :developer do
+    # Create the main branch
+    run 'touch .gitignore ; git add .gitignore ; git commit -m "Initial commit"; git push -u origin master'
+    run 'git checkout -b main master ; git push -u origin main'
+
+    # Fetch the default branch, delete master
     run 'git fetch'
     run 'git push origin :master'
     run 'git branch -d master'
   end
 
-  go_to_repository :developer
+  # memoize environment by saving directory contents
+  FileUtils.cp_r "#{REPOSITORY_BASE}/.", MEMOIZED_REPOSITORY_BASE
+
+  $memoization_complete = true
```

Comment:
Fails to work. Not too sure what the class hierarchy looks like here. The global var was suggested on cucumber's docs.